README.md

```diff
@@ -1,2 +1,34 @@
 # potential-fortnight
 Top secret game, feel free to contribute.
+
+Style Guide
+-----------
+This project will defer to [idiomatic.js](https://github.com/rwaldron/idiomatic.js/), with some elaborations and exceptions below.
+
+* Use 4 space indentation, no tabs
+* Always use spacing between operators (i.e. `2 + 2` not `2+2`)
+* Don't use additional whitespace inside parens (i.e. `(2 + 2)` not `( 2 + 2 )`)
+* Parens should stay on the same line for conditionals and else clauses should share a line with closing parens i.e.
```

Comment:
I think `(` are parentheses, `{` are called braces, if so then change the first word in this line.